### DISCUSSION TITLE
Where should I go to open a software in the sandbox?

### DISCUSSION THREAD

**Discussion by ronecc1** [sandboxie-plus/Sandboxie#2821] [Documents and notes]
I get the idea of sandboxie, but even after reading all the blurb, I am still at a loss as to how to use it. I have downloaded a piece of software, but have not opened it. It is sat in my downloads box. I open sandbox DefaultBox>Run Sandbox>, but where do I go to load the downloaded item. If I unpack and install, their could be a virus, so how do I open it in sandbox?
Sorry, I think I have sussed it!

**Comment by offhub** [comment]
DefaultBox > Run Sandboxed > Run Any Program > Select the software

OR

Right click on the file and select "Run Sandboxed" and then choose a sandbox from the menu.

**Reply by ronecc1** [reply]
Thanks offhub, I sorted it just after posting. I had tried a few things, but just got error popups. I have now got the software running and testing it. Trouble is it is a trial period so I cannot export or save the results, but at least I now know how the software works and I will not be paying the £250 they want.

**Reply by isaak654** [reply]
For the sake of completeness:

1. According to the [Known Conflicts](https://sandboxie-plus.github.io/sandboxie-docs/Content/KnownConflicts.html) documentation, not all programs can be installed or run inside Sandboxie.

2. It is still possible to save the Sandboxie messages (provided as error popups) in a separate log file: 
[MessagesFromSandboxie.html#log-messages-to-a-file](https://sandboxie-plus.github.io/sandboxie-docs/Content/MessagesFromSandboxie.html#log-messages-to-a-file)

3. Only a limited number of Sandboxie features requires a supporter certificate for their unlimited use over the 5-minute time limit. If you remove them, you should still be able to use Sandboxie for free (except for [nag screens](https://github.com/sandboxie-plus/Sandboxie/discussions/2107) that can be skipped). The last point does not apply if enterprise use has been selected in the Sandboxie Plus setup wizard.

4. If you contribute to the repository, you could get a supporter certificate for free: [CONTRIBUTING.md#non-monetary-contributions](https://github.com/sandboxie-plus/Sandboxie/blob/master/CONTRIBUTING.md#non-monetary-contributions)

**Comment by sergeevabc** [comment]
Aggrrhh! How do I get rid of “Run Sandboxed” window?
There is only one DefaultBox, and I don't like “hold Ctrl to skip” workaround.
Surely there must be an option somewhere in .INI or deep in the GUI, or is there not?

![2024-0202-0537 sandboxie 567 skip window](https://github.com/sandboxie-plus/Sandboxie/assets/373718/8e9b88ba-bf3f-4132-a209-d0c701010e47)

**Reply by offhub** [reply]
This setting is available in the Sandboxie Plus version.

`Sandboxie Plus > Options > Shell Integration > Run Sandbxoed - Actions: Always use DefaultBox
`

**Reply by sergeevabc** [reply]
I use Classic version (5.67.7) and have no plans to replace it with Plus. Did it once then rolled back. Could you be so kind to implement this option there (.INI is fine if changing GUI takes more time)? Seriously, that window drives me crazy, it's not good for the health.

**Reply by isaak654** [reply]
According to the README file:

> Sandboxie Classic has the old no longer developed MFC-based UI, hence it lacks native interface support for Plus features. 

So far, no active developer has come forward in over three years.

**Reply by sergeevabc** [reply]
Sorry, but why do classic releases have version changes then? Is it just like that and no changes are made at all?

**Reply by isaak654** [reply]
> Sorry, but why do classic releases have version changes then?

For simplicity and because there are common components between Classic and Plus as in commit 027e08a.

For more details, do not hesitate to consult the source code and documentation.
